soroban-sdk/src/env.rs

```diff
@@ -423,13 +419,14 @@ impl Env {
         contract_id
     }
 
-    /// Register a contract in a WASM file with the [Env] for testing.
+    /// Install the contract WASM code to the [Env] for testing.
     ///
-    /// Passing a contract ID for the first arguments registers the contract
-    /// with that contract ID. Providing `None` causes a random ID to be
-    /// assigned to the contract.
+    /// Returns the identifier of the installed code that can be then used for
```

Comment:
We refer to the value returned as the "identifier" here, but in other places in this PR it is referred to as the "wasm_hash". We need to stick with one term for the thing otherwise it will be somewhat confusing to a developer. I suggest we stick with hash because we should expect people to be able to hash their contract file independently to find the value so it seems odd to abstract the concept. If it's not just a simple hash of the wasm though, identifier is better.

Reply:
Updated to use hash everywhere.